tutor/src/screens/course-settings/timezone.jsx

```diff
@@ -172,7 +171,7 @@ class SetTimezone extends React.Component {
               defaultValue={this.props.course.time_zone}
               onChange={val => this.course_timezone = val}
               validate={this.validate}
-              autofocus={true} />
```

Comment:
Just making sure removing the focus on SetTimezone was intentional.

Reply:
yeah, thanks for checking.  the `autofocus` here was never actually read by the component that was rendered here.  Instead it was just setting it to true itself.  I removed it since there was no reason to set it twice.